modules/ROOT/pages/about-components.adoc

```diff
@@ -1,57 +1,176 @@
+* Batch Aggregator
+* Batch Job
+* Batch Step
 
-Connectors and Schedulers trigger flows by passing in an event. Other components then transform and process the event though a sequence of linked steps.
+See xref:batch-processing-concept.adoc[Batch Processors].
+
+== Components
 
-These event processing components include individual event processors like a Logger or Transform component, or other outbound connectors that request additional data to be merged with the existing event later in the flow.
+These Core components perform a variety of tasks:
 
-To help combine and transform data, event processors can be grouped together to perform other common processing tasks. These groupings are categorized as Routers or Scopes.
+* xref:business-events-custom.adoc[Custom Business Events]: For collecting information about about flows and message processors that handle your business transactions. See also, xref:business-events.adoc[Business Events].
+* xref:dynamic-evaluate-component-reference.adoc[Dynamic Evaluate]: For dynamically selecting a script, instead of hardcoding it through the Transform Message Component.
+* xref:flowref-about.adoc[Flow Reference]: For routing the Mule event to another
+flow or subflow (and back) within a Mule app.
+* xref:logger-component-reference.adoc[Logger]: For logging important information about
+your Mule app, such as error messages and status notifications.
+* xref:parse-template-reference.adoc[Parse Template]: For processing a template
+and obtaining a result.
+* xref:transform-component-about.adoc[Transform Message]: For converting input
+data to a new output structure or format.
 
-A Scope (blue color in Design Center) groups a sequence of event processors together to apply some programming behavior to that isolated sequence of event processors.
+== Endpoints
 
-For example, a Try scope lets you isolate and handle any errors that occur in a particular sequence of flows. So, you might wrap an outbound HTTP Request connector (and perhaps some other surrounding components before and after the HTTP Request connector, such as a Transform Message component or a Logging component) in a Try scope, so if an error results, you can apply logging or compensation logic specific to that grouping of components.
+An endpoint is a Core component that initiates (or triggers) processing in a
+Mule flow. The xref:scheduler-concept.adoc[Scheduler] serves as an endpoint. It triggers a flow to start at a configurable interval.
 
-A For Each scope takes a collection of data, usually extracted from the current input event, and applies the same sequence of event processors to every item in the collection. For example, a For Each scope might be used to process each individual row returned from a database query or each individual line from a CSV file.
+Note that some connectors provide listeners that serve as endpoints. Listeners
```

Comment:
Exactly, this is actually a source as the ones connectors provide.

Reply:
Maybe we should request the Studio team to change "Endpoints" to "Source" or "Event Source".